server/http/restapi/asset.go

```diff
@@ -0,0 +1,45 @@
+package restapi
+
+import (
+	"net/http"
+
+	"github.com/mattermost/mattermost-plugin-apps/server/api"
+	"github.com/mattermost/mattermost-plugin-apps/server/utils/httputils"
+	"github.com/pkg/errors"
+)
+
+func (a *restapi) assetGet(w http.ResponseWriter, req *http.Request) {
```

Comment:
1/5 s/assetGet/handleGetAsset to be in sync with the other non-KV router mappings.